core/timeline_inc.php

```diff
@@ -67,20 +66,30 @@
 					echo ' <a class="btn btn-primary btn-xs btn-white btn-round" href="my_view_page.php?days=' .
 						$t_next_days . '">' . lang_get( 'next' ) . '</a>';
 				}
+				echo '</div>';
+?>
+			</div>
+		</div>
 
-			echo '</div></div></div><div class="widget-main no-padding"><div class="profile-feed">';
-
-	echo '</div></div>';
+		<div class="widget-main no-padding">
+			<div class="profile-feed">
+			</div>
+		</div>
```

Comment:
@syncguru are these 2 divs really useful ? It looks like they are not doing anything (I don't see any differences in display if I remove them), and are printed later on (by _timeline_print_events()_ call), so I'm wondering if they are maybe a leftover of older code.